Modules/_imp.c

```diff
@@ -0,0 +1,29 @@
+#define PY_SSIZE_T_CLEAN
+#include "Python.h"
+#include "clinic/_imp.c.h"
+
+static PyObject *
+is_lazy_key_impl(PyObject *module, PyObject *dict, PyObject *key)
+{
+    int res = PyDict_IsLazyKey(dict, key);
+    return Py_BuildValue("i", res);
+}
+
+static PyMethodDef LazyDebugMethods[] = {
+    IS_LAZY_KEY_METHODDEF
+    {NULL, NULL, 0, NULL}
+};
+
+static struct PyModuleDef lazydebugmodule = {
+    PyModuleDef_HEAD_INIT,
+    "_lazydebug",
```

Comment:
we should have this under `_importlib`

related to the other discussion we had - `_imp.c` is not the appropriate name for this module - let's move it to `_importlib.c` and export it as `_importlib`

Reply:
Yes, this also would be helpful for the following tasks!